webapp/src/Controller/Jury/ConfigController.php

```diff
@@ -75,6 +75,12 @@ public function indexAction(EventLogService $eventLogService, Request $request):
                         }
                     }
                     $data[substr($key, strlen('config_'))] = $valueToUse;
+                    if (strpos($key, 'config_lazy_eval_results') === 0 && $value !== '2') {
```

Comment:
Why can't we check the full key here? i.e. `if ($key === 'config_lazy_eval_results')`?